### ISSUE TITLE
how to use same identifier for  different directory paths in ejs with different static assest folder  

### ISSUE THREAD

**Issue by matt212** [fastify/point-of-view#334]
### Prerequisites

- [X] I have written a descriptive issue title
- [X] I have searched existing issues to ensure the issue has not already been raised

### Issue

this is my ejs 

```
<head>
   
     <%- include('../../public/components/headercontent.html'); %>
     
</head>
```

and below is my app/server.js file configuration

```
fastify.register(require("point-of-view"), {
  engine: {
    ejs: require("ejs"),
  },
  root: path.join(__dirname, "../views"),
});


fastify.register(require("fastify-static"), {
  root: path.join(__dirname, "../") + "/public/",
});
```

so the above works as expected my problem statement is below 
i have two static assets folder 

- public 
- public-release

what is expected scenario is 

I want to use same public name but path can be different 
something like  
case scenario 1
```
fastify.register(require("fastify-static"), {
  root: path.join(__dirname, "../") + "/**public-release**/",
  nameused:'public'
});
```
case scenario 2
```
fastify.register(require("fastify-static"), {
  root: path.join(__dirname, "../") + "/**public**/",
  nameused:'public'
});
```

is there a way I can achieve this ?

**Objective :** 

1.i want to switch dynamically my js and css for home grown release /bundling system 
and without touching my existing codebase of linking them
 
2.so the idea here is to just the 
update the point where static assests are served and change them through custom yarn or npm run scripts

3. **for eg:- when i passed say dev the static assets should serve from 'public ' folder**

   **and when i passed say prod the static assets should serve from 'public-release' folder**

**Comment by matt212** [comment]
So , there is middle ground approach which I implemented for my boilerplate

[Github Repo: Node.js Postgresql fastify](https://github.com/matt212/Nodejs_Postgresql_VanillaJS_Fastify)

TLDR;
<img width="960" alt="nodejs-web-app-Env-pt1" src="https://user-images.githubusercontent.com/1698139/182035909-e05d94dc-0a4e-4b71-981f-b50ee0024121.png">

<img width="960" alt="nodejs-web-app-Env-pt2" src="https://user-images.githubusercontent.com/1698139/182035918-485e35c7-d19b-46a6-b9f7-7dbcd1300357.png">

![how-to-release-node-app](https://user-images.githubusercontent.com/1698139/182035929-7646209f-65ab-4934-8f65-0760391874e0.png)
